api/src/routes/pipelines.rs

```diff
@@ -191,6 +200,12 @@ pub struct ReadPipelinesResponse {
     pub pipelines: Vec<ReadPipelineResponse>,
 }
 
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+pub struct SwapImageRequest {
```

Comment:
Rename `swap` to `update` everywhere? Swap has connotations of exchanging two things but we are just overwriting the old image.